test/values.test.js

```diff
@@ -255,6 +255,42 @@ describe('Values', () => {
     )
   })
 
+  test('pretty print Match', () => {
+    assertPrint(
+      new Query(q.Lambda('_', q.Match(q.Index('idx')))),
+      'Query(Lambda("_", Match(Index("idx"))))'
+    )
+
+    assertPrint(
+      new Query(q.Lambda('_', q.Match(q.Index('idx'), 'str'))),
+      'Query(Lambda("_", Match(Index("idx"), "str")))'
+    )
+
+    assertPrint(
+      new Query(q.Lambda('_', q.Match(q.Index('idx'), 'str', 10))),
+      'Query(Lambda("_", Match(Index("idx"), "str", 10)))'
+    )
+
+    assertPrint(
+      new Query(q.Lambda('_', q.Match(q.Index('idx'), ['str', 10]))),
+      'Query(Lambda("_", Match(Index("idx"), "str", 10)))'
```

Comment:
I was expecting `"str", 10` to be renderized as `["str", 10]`, we have a condition on the code to do this